src/helix-ui/elements/HXAccordionElement.js

```diff
@@ -27,12 +27,17 @@ export class HXAccordionElement extends HXElement {
 
     $onConnect () {
         this.$upgradeProperty('currentPanel');
+        // FIXME: panels are not connected at this point
         this.panels.forEach(panel => {
             panel.addEventListener('open', this._onPanelOpen);
         });
+
+        // FIXME: initialize on connect
     }
 
     $onDisconnect () {
+        // FIXME: panels may not be present on disconnect
+        // (react disconnects children before the parent)
```

Comment:
NOTE: Need to verify React behavior

Reply:
* Parent React components will have their `componentWillUnmount()` called before any child `disconnectedCallback()` and `componentWillUnmount()` is called. 
    * Disconnect/Mount from the outside-in.
* Children are always present on disconnect.